hpx/runtime/actions/continuation.hpp

```diff
@@ -153,10 +155,10 @@ namespace hpx { namespace actions
         virtual ~continuation() {}
 
         //
-        virtual void trigger() const;
+        virtual void trigger();
 
         template <typename Arg0>
-        inline void trigger(Arg0 && arg0) const;
+        inline void trigger(Arg0 && arg0);
 
         //
         virtual void trigger_error(boost::exception_ptr const& e) const;
```

Comment:
trigger might move something from `this`, `trigger_error()` doesn't. That's why I chose to keep it const because it doesn't need to be non-const. However if you think it would be better for API symmetry, that can be changed as well.